src/join.sol

```diff
@@ -79,35 +85,52 @@ contract GemJoin {
     uint    public dec;
     uint    public live;  // Active Flag
 
+    // Events
+    event Rely(address indexed usr);
+    event Deny(address indexed usr);
+    event Join(address indexed urn, uint256 wad);
```

Comment:
just wondering - are we sure we want the event parameter (`urn`) to have a different name than the function parameter (`usr`)?

Reply:
This should be `usr`.